src/Application.js

```diff
@@ -26,6 +26,7 @@ const completionPercentage = 'completionPercentage';
 const speedScale = 'speedScale';
 const timersScale = 'timersScale';
 const inputCount = 'inputCount';
+const fullscreen = 'fullscreen';
```

Comment:
I would update this to be `fullScreen` not `fullscreen` just a little more readable. Make sure it update it everywhere.